hive-event-listeners/apiary-gluesync-listener/src/main/java/com/expediagroup/apiary/extensions/gluesync/cli/GlueSyncCli.java

```diff
@@ -139,8 +141,8 @@ public void syncAll(CommandLine cmd) throws TException {
     }
   }
 
-  private void syncTable(String dbName, String tableName, boolean deleteGluePartitions, boolean verbose)
-      throws TException {
+  private void syncTable(String dbName, String tableName, boolean deleteGluePartitions, boolean syncOnlyViews,
```

Comment:
Not a big deal, but if we need to keep adding options to this method, we should consider adding a `SyncTableOptions` class that encapsulates `deleteGluePartitions`, `syncOnlyViews`, and `verbose`.